Task 2/Task 2/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+class Program
+{
+    static void Main()
+    {
+
+        Console.WriteLine("Введите координаты центра квадрата (x, y):");
+        Console.Write("x: ");
+        double centerX = Convert.ToDouble(Console.ReadLine());
+        Console.Write("y: ");
+        double centerY = Convert.ToDouble(Console.ReadLine());
+
+
+        Console.WriteLine("Введите координаты одной из вершин квадрата (x, y):");
+        Console.Write("x: ");
+        double vertexX = Convert.ToDouble(Console.ReadLine());
+        Console.Write("y: ");
+        double vertexY = Convert.ToDouble(Console.ReadLine());
```

Comment:
Лучше не использовать методы класса `Convert `без особой причины.
```suggestion
        double centerX = double.Parse(Console.ReadLine());
        Console.Write("y: ");
        double centerY = double.Parse(Console.ReadLine());

        Console.WriteLine("Введите координаты одной из вершин квадрата (x, y):");
        Console.Write("x: ");
        double vertexX = double.Parse(Console.ReadLine());
        Console.Write("y: ");
        double vertexY = double.Parse(Console.ReadLine());
```